src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -503,36 +503,81 @@ public string DictionaryLink
         {
             get
             {
-                string finalLink = Configurations.DynamoDictionary + "#/";
-                if (category == null || category == "")
-                {
-                    return Configurations.DynamoDictionary; // if there is no category, return the link to home page
-                }
+                dictionaryLink = dictionaryLink ?? Configurations.DynamoDictionary;
+                return dictionaryLink;
+            }
+            set
+            {
+                dictionaryLink = value;
+            }
+        }
+
+        private string dictionaryLink;
+
+        public string GetDictionaryLinkFromLibrary(LibraryServices libraryServices)
+        {
+            string finalLink = Configurations.DynamoDictionary + "#/";
+            if (IsCustomFunction)
+            {
+                return ""; // If it is not a core or Revit function, do not display the dictionary link
```

Comment:
@aparajit-pratap @yeexinc Please leave the Revit links as they are. If there is an invalid link on the dictionary it will redirect to a coming soon page.